cmd/nerdctl/compose_exec_linux_test.go

```diff
@@ -206,8 +194,15 @@ services:
 	unbuffer := []string{"unbuffer"}
 	base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(), "exec", "svc0", "stty").AssertOutContains(sttyPartialOutput)             // `-it`
 	base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(), "exec", "-i=false", "svc0", "stty").AssertOutContains(sttyPartialOutput) // `-t`
-	base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(), "exec", "-t=false", "svc0", "stty").AssertFail()                         // `-i`
-	base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(), "exec", "-i=false", "-t=false", "svc0", "stty").AssertFail()
+
+	// The error output is different with docker
```

Comment:
Can we fix this?

Reply:
No. They even don't have the same exit code. We might have to deal with this one separately.

Reply:
Why can't we make the exit code same?

Reply:
If it is difficult for now it should be just marked as DockerIncompatible and we can revisit it later

Reply:
This change has been reverted.